src/Note.ts

```diff
@@ -60,6 +66,7 @@ class Note implements PlayableInterface {
     }, this.duration * 1000);
   }
 
+
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏One too many lines here

Reply:
🙃